afl/src/bin/cargo-afl.rs

```diff
@@ -382,7 +418,15 @@ mod tests {
     }
 
     const SUBCOMMANDS: &[&str] = &[
-        "analyze", "cmin", "fuzz", "gotcpu", "plot", "showmap", "tmin", "whatsup",
+        "analyze",
+        "cmin",
+        "fuzz",
+        "gotcpu",
+        "system-config",
+        "plot",
+        "showmap",
+        "tmin",
+        "whatsup",
```

Comment:
How did you choose this order? In the absence of some other reason, I would prefer to default to alphabetical order, like before.

Also, could you please ensure that the order in which the subcommands are listed above matches how they are listed here?